tests/x11/ibus/ibus_test_kr.pm

```diff
@@ -0,0 +1,72 @@
+# SUSE's openQA tests
+#
+# Author: Gao Zhiyuan <zgao@suse.com>
+# Copyright © SUSE LLC
```

Comment:
Could you please use the common format we have in other files with the copyright year and no author? We have the author information in git anyway :)

Reply:
Sorry for the trouble. 
[New PR fixes licenses](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6330)